code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -569,6 +573,11 @@
 	repeatable = TRUE
 	var/list/spawn_locs = list()
 
+/datum/dynamic_ruleset/midround/from_ghosts/nightmare/acceptable(population=0, threat=0)
+	if (!SSmapping.empty_space)
```

Comment:
whats this

Reply:
I... this should be in execute, I believe